### ISSUE TITLE
Installation Difficulties

### ISSUE THREAD

**Issue by PeterMcLennan** [RupertAvery/DiffusionToolkit#215]
Installed the latest version of .NET as instructed.
Validated .NET 8.0.1 appears to have installed correctly
Unzipped installer folder.
Disabled SmartScreen Filter in Windows 10 64 bit so Diffusion.Toolkit.exe will run.
On running Diffusion.Toolkit.exe got the warning from SmartScreen and chose "run anyway"
A brief flash of what looked like a small window opening and disappearing.
Then, nothing.  No sign of the app.

Any advice?

**Comment by RupertAvery** [comment]
The required version is the .NET 6 Desktop Runtime. 

https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.26-windows-x64-installer

Also, did you extract all of the files into a folder?

**Comment by PeterMcLennan** [comment]
Ah.  So I installed the wrong version of .NET?  
Can I install your linked version without uninstalling my mistake?
I unzipped it all into a folder in the root directory of C:

**Comment by RupertAvery** [comment]
Yes, should be no problem. .NET versions can exist alongside each other, no problem. Let me know if this fixes your problem, or if you still have issues.

**Comment by PeterMcLennan** [comment]
That did it!  Thanks so much for your prompt responses.  Gives me great confidence going fwd.  

I've not attempted anything yet, I just got past the splash screen and to the main interface.  

In the past, compatibility with Fooocus was missing.  I read that it does now support RuinedFooocus, so I'm ready to try.  I am a Fooocus and Invoke user, but probably willing to change either or both if it means I can regain control over my outputs dirs.  :)

**Comment by PeterMcLennan** [comment]
Holy &$#! what a great program!  An order of magnitude faster than File Explorer.  Took about 30 seconds to add 10K images to the database.  And the database is tiny!  
Amazing work.  And an welcome timesaver for us with clogged-up output folders.

Not seeing any creation metadata from my Fooocus images, but that might be operator error just like my install.

THANK YOU! :)

**Comment by RupertAvery** [comment]
Fooocus currently outputs metadata to an html file so its not embedded in the image like others. Although I believe there is support being added for A1111-style metadata.

**Comment by PeterMcLennan** [comment]
Excellent!  Thanks.
Does RuinedFooocus embed the metadata?

**Comment by RupertAvery** [comment]
Yes, it does embed the metadata and DiffusionToolkit does support reading it.
